drivers/charger/charger_bq25180.c

```diff
@@ -87,6 +91,32 @@ static uint32_t bq25180_ichg_to_ma(uint8_t ichg)
 	return (ichg - 31) * 10 + 40;
 }
 
+/* VBATREG= 3.5V + VBATREG_CODE * 10mV
+ * Maximum programmable voltage = 4.65V
+ *
+ * Return: value between 0 and 127, negative on error.
+ */
+static int bq25180_mv_to_vbatreg(uint32_t voltage_mv, uint8_t *vbat)
+{
+	if (!IN_RANGE(voltage_mv, BQ25180_VOLTAGE_MIN_MV, BQ25180_VOLTAGE_MAX_MV)) {
+		LOG_WRN("charging voltage out of range: %dmV, "
+			"clamping to the nearest limit",
+			voltage_mv);
+	}
+	voltage_mv = CLAMP(voltage_mv, BQ25180_VOLTAGE_MIN_MV, BQ25180_VOLTAGE_MAX_MV);
+
+	*vbat = (voltage_mv - 3500) / 10;
```

Comment:
Please `#define` the magic numbers here. With those magic numbers defined, you could honestly drop the comment block for the function.

Reply:
Done